src/sst/core/simulation.cc

```diff
@@ -1112,6 +1112,22 @@ Simulation_impl::registerClock(const UnitAlgebra& freq, Clock::HandlerBase* hand
     return registerClock(tcFreq, handler, priority);
 }
 
+TimeConverter*
+Simulation_impl::registerClock(TimeConverter& tc_freq, Clock::HandlerBase* handler, int priority)
```

Comment:
Should be TimeConverter instead of TimeConverter& for this function and reregisterClock(), getNextClockCycle() and unregisterClock()